src/middleware/router.js

```diff
@@ -281,16 +281,10 @@ function sendRequestToRoutes (ctx, routes, next) {
 
     Promise.all(promises).then(() => {
       logger.info(`All routes completed for transaction: ${ctx.transactionId}`)
+
       // Set the final status of the transaction
-/*
-      messageStore.setFinalStatus(ctx, err => {
-        if (err) {
-          logger.error(`Setting final status failed for transaction: ${ctx.transactionId}`, err)
-          return
-        }
-        logger.debug(`Set final status for transaction: ${ctx.transactionId}`)
-      })
- */
+      setTransactionFinalStatus(ctx)
+
       // TODO: OHM-694 Uncomment when secondary routes are supported
```

Comment:
I think we need to also look at bringing this code back in now that the secondary routes are back in again